site/docs/how-tos/operator.md

```diff
@@ -0,0 +1,309 @@
+operatorhubio-catalog-7z5wh         1/1     Running   0          1m
+packageserver-854879f5cf-49z64      1/1     Running   0          1m
+packageserver-854879f5cf-r7kzz      1/1     Running   0          1m
+```
+
+#### Installing the Hazelcast Jet Operator
+
+Install the Hazelcast Jet Operator by running the following command:
+
+```bash
+kubectl create -f https://operatorhub.io/install/hazelcast-jet-operator.yaml
+```
+
+Hazelcast Jet Operator will be installed in the `operators` namespace
+and will be usable from all namespaces in the cluster.
+
+After install, watch your operator come up using the following command.
+
+```bash
+$ kubectl get csv -n operators
+NAME                            DISPLAY                  VERSION   REPLACES   PHASE
+hazelcast-jet-operator.v0.1.0   Hazelcast Jet Operator   0.1.0                Succeeded
+```
+
+### Manual Installation
+
+If you'd like to manually install the Hazelcast Jet Operator without
+installing the OLM, please follow the README on [Hazelcast Jet Operator](https://github.com/hazelcast/hazelcast-jet-operator/tree/master/hazelcast-jet-operator)
+GitHub repository.
+
+## Creating a Hazelcast Jet Cluster
+
+After successfully installing the Hazelcast Jet Operator to your cluster
+we can now create Hazelcast Jet clusters with the `HazelcastJet` custom
+resource.
+
+Create a file name `hazelcast-jet-.yaml` with the contents below:
+
+```yaml
+apiVersion: hazelcast.com/v1alpha1
+kind: HazelcastJet
+metadata:
+  name: jet-cluster
+spec:
+  cluster:
+    memberCount: 2
+  securityContext:
+    runAsUser: ""
+    runAsGroup: ""
+    fsGroup: ""
+```
+
+**Note:** You can use [this
+file](https://github.com/hazelcast/hazelcast-jet-operator/blob/master/hazelcast-jet-operator/hazelcast-jet-full.yaml)
+for configuration options reference.
+
+>### Setting the License Key for Management Center (Optional)
+>
+>If you do have an
+>active license subscription for Hazelcast Jet Management Center, use
+>the configuration below to start your cluster. You can
+>apply for a 30-day trial if you don't have one from [Hazelcast Website](https://hazelcast.com/get-started/#deploymenttype-jet)
+>. This step is optional and only for Hazelcast Jet Management Center.
+> Hazelcast Jet doesn't require a license key. You can also skip the
+> deployment of Management Center altogether by setting the `managementcenter.enabled`
+> property to `false`.
+>
+>```yaml
+>apiVersion: hazelcast.com/v1alpha1
+>kind: HazelcastJet
+>metadata:
+>  name: jet-cluster
+>spec:
+>  cluster:
+>    memberCount: 2
+>  securityContext:
+>    runAsUser: ""
+>    runAsGroup: ""
+>    fsGroup: ""
+>  managementcenter:
+>    licenseKey: <YOUR_LICENSE_KEY>
+>```
+
+Start Hazelcast Jet cluster with the following command.
+
+```bash
+kubectl apply -f hazelcast-jet.yaml
+```
+
+Your Hazelcast Jet cluster (together with Hazelcast Jet Management Center)
+should be created.
+
+```bash
+$ kubectl get all
```

Comment:
I get quite different output here (more services etc..)

```
kubectl get all                             
NAME                                                               READY   STATUS    RESTARTS   AGE
pod/hazelcast-jet-operator-86d6bb6b78-z88r9                        1/1     Running   1          20d
pod/jet-cluster-hazelcast-jet-0                                    1/1     Running   0          2m27s
pod/jet-cluster-hazelcast-jet-1                                    1/1     Running   0          3m17s
pod/jet-cluster-hazelcast-jet-management-center-86df845c89-864ds   1/1     Running   0          3m22s

NAME                                                  TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)                        AGE
service/hazelcast-jet-operator-metrics                ClusterIP      10.105.209.34   <none>        8686/TCP,8383/TCP              20d
service/jet-cluster-hazelcast-jet                     ClusterIP      None            <none>        5701/TCP                       20d
service/jet-cluster-hazelcast-jet-management-center   LoadBalancer   10.102.196.72   <pending>     8081:32124/TCP,443:30167/TCP   20d
service/kubernetes                                    ClusterIP      10.96.0.1       <none>        443/TCP                        20d

NAME                                                          READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/hazelcast-jet-operator                        1/1     1            1           20d
deployment.apps/jet-cluster-hazelcast-jet-management-center   1/1     1            1           20d

NAME                                                                     DESIRED   CURRENT   READY   AGE
replicaset.apps/hazelcast-jet-operator-86d6bb6b78                        1         1         1       20d
replicaset.apps/jet-cluster-hazelcast-jet-management-center-6c567894d4   0         0         0       20d
replicaset.apps/jet-cluster-hazelcast-jet-management-center-86df845c89   1         1         1       3m22s

NAME                                         READY   AGE
statefulset.apps/jet-cluster-hazelcast-jet   2/2     20d
```

Reply:
Hmm, now I noticed `AGE 20d`  so I suppose I didn't clean up when I tried the steps in readme.

Reply:
Yes, this was old state, but the difference between `jet-hazelcast-jet-1` and `jet-cluster-hazelcast-jet-1` is there anyway.

Reply:
I think you'd have an Operator and a cluster deployed a while ago..